### PR TITLE
Update default for checkout flag

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Addresses https://github.com/blackbaud/skyux2-docs/issues/981.

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-checkout-flag-default-072820](https://developer.blackbaud.com/skyux?_sp.spa=pr-checkout-flag-default-072820)

File: src/stache/includes/cli/checkout.html [hunk 1]
```diff
@@ -1 +1 @@
-<sky-code>--checkout</sky-code> — Specifies the branch to clone from the template. The default is <sky-code>master</sky-code>.
+<sky-code>--checkout</sky-code> — Specifies the branch to clone from the template. The default is the versioned branch with the <sky-code>[MAJOR_VERSION].x.x</sky-code> format that is required in the template's repo.
```
Comment (Blackbaud-BobbyEarl) [inline src/stache/includes/cli/checkout.html:1]: This sentence was a bit hard to read.  We also need to mention it's the `MAJOR_VERSION` associated the SKY UX CLI installed.  Perhaps something like:

The default branch is controlled by the version of the SKY UX CLI installed using the following format: `[SKYUX CLI Major Version].x.x`.

Comment (blackbaud-johnly) [reply src/stache/includes/cli/checkout.html]: Updated this and the corresponding note at the bottom of the page.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-checkout-flag-default-072820](https://developer.blackbaud.com/skyux?_sp.spa=pr-checkout-flag-default-072820)

**Pull request merged**